src/components/Copyable.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+    <div class="copyable" :class="{ copied }" @click="copy">
+        <div class="background"></div>
+        <slot></slot>
+    </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Prop } from 'vue-property-decorator';
+import { Clipboard } from '@nimiq/utils';
+
+/**
+ * **Copyable**
+ *
+ * Copyable can be used to make a click on one or more elements copy content to the Clipboard with visual feedback.
+ * By default the children's contents are copied to the Clipboard. Alternatively, a specific text to be copied can be
+ * provided.
+ *
+ * Props:
+ *
+ * **text** {string} [optional] - A specific text to be copied to the clipboard
+ */
+@Component
+export default class Copyable extends Vue {
+    @Prop(String)
+    public text?: string;
+
+    public copy() {
+        // Note that when iterating over childNodes, pseudo elements are excluded.
+        const text = this.text || [ ...this.$el.childNodes ].reduce((concatenated, node) => {
+            let nodeText = '';
+            if (node.nodeType === Node.TEXT_NODE) {
+                nodeText = node.textContent.trim();
+            } else if (node.nodeType === Node.ELEMENT_NODE) {
+                // use innerText as it better represents the visible content than textContent
+                nodeText = (node as HTMLElement).innerText;
+            }
+
+            if (concatenated && nodeText) {
+                concatenated += '\n';
+            }
+            concatenated += nodeText;
+            return concatenated;
+        }, '');
+        Clipboard.copy(text);
+
+        window.clearTimeout(this._copiedResetTimeout);
+        this.copied = true;
+        this._copiedResetTimeout = window.setTimeout(() => {
+            this.copied = false;
+        }, 1800);
+    }
+
+    private copied: boolean = false;
+    private _copiedResetTimeout: number | null = null;
+}
+</script>
+
+<style scoped>
+    .copyable {
+        position: relative;
+        padding: 1.5rem;
+        cursor: pointer;
+        margin-bottom: 5.5rem;
+    }
+
+    .background {
+        position: absolute;
+        left: 0;
+        top: 0;
+        width: 100%;
+        height: 100%;
+        background-image: var(--nimiq-blue-bg);
+        border-radius: .5rem;
+        opacity: 0;
+        transition: opacity .3s var(--nimiq-ease);
+    }
+
+    .copyable:hover .background,
+    .copyable.copied .background {
+        opacity: .05;
+    }
+
+    .copyable::before,
+    .copyable::after {
+        position: absolute;
+        left: 50%;
+        top: 100%;
+        /* When label is not visible set height to 0 to avoid it being hoverable. Not using pointer-events:none as the
```

Comment:
```suggestion
        /* When label is not visible, set height to 0 to avoid it being hoverable. Not using pointer-events:none here as the
```

Reply:
This comment had since been removed as the css logic changed.